api/v6/api.go

```diff
@@ -68,7 +68,7 @@ type NotDeprecated interface {
 	Export(context.Context) ([]byte, error)
 
 	// v6
-	ListServices(ctx context.Context, namespace string) ([]ControllerStatus, error)
+	ListControllers(ctx context.Context, namespace string) ([]ControllerStatus, error)
```

Comment:
The API methods get called by RPC via reflection. Annoyingly.

Reply:
Where does that happen?

Reply:
I recall renaming the strings making the calls

Reply:
That ... probably is good enough ... though the client part of the remote package runs in weave cloud, and we have to be sure that changing the name won't break compatibility.